README.md

```diff
@@ -32,14 +32,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+The Document Object Model (DOM) is an object representation of html elements in a webpage.
+
 2. What is an event?
 
+All user interactions with the webpage.
+
 3. What is an event listener?
 
+An event listener is when we select a element to listen to an event and as a result it run a callback function.
+
 4. Why would we convert a NodeList into an Array?
 
+This allowed the Developer access to array method like .forEach.
```

Comment:
You're correct that this allows developers to access array methods. However, we can iterate a nodeList using *forEach*.